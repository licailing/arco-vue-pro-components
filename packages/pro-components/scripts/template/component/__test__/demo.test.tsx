import demoTest from '../../../../scripts/demo-test';
import BasicDemo from '../__demo__/basic.vue';

demoTest({ BasicDemo });
