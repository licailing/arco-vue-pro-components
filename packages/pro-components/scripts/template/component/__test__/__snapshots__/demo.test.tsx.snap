// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Demo Test: render [BasicDemo] correctly 1`] = `"<templatecomponent>Demo Basic</templatecomponent>"`;
